.travis.yml

```diff
@@ -9,12 +9,16 @@ julia:
     - nightly
 notifications:
     email: false
-    
+
 branches:
     only:
         - master
         - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
 
+jobs:
+    allow_failures:
+        - julia: nightly
```

Comment:
```suggestion
        - julia: nightly
          os: windows
```

Reply:
I'm usually of the opinion that nightly can fail on any OS it wants, but this is fine too.